components/templates/src/global_fns/load_data.rs

```diff
@@ -291,7 +291,16 @@ fn load_csv(csv_data: String) -> Result<Value> {
         let mut records_array: Vec<Value> = Vec::new();
 
         for result in records {
-            let record = result.unwrap();
+            let record = match result {
+                Ok(r) => r,
+                Err(e) => {
+                    return Err(tera::Error::chain(
+                        String::from("Error encountered when parsing csv records"),
+                        e,
+                    ));
+                }
+            };
```

Comment:
you want to use zola errors, not tera ;)

I believe you can replace this line by 

```rust
let record = result..map_err(|e| Error::chain("Error encountered when parsing csv records", e))?;
```

Reply:
Sure, sorry about the oversight. I'll take a look soon and update PR.

Reply:
Sorry actually, I just realised this is a Tera function so using the Tera error makes sense!